src/auth.ts

```diff
@@ -66,21 +66,22 @@ async function authorize(useLocalhost: boolean, writeToOwnKey: boolean) {
 /**
  * Loads the Apps Script API credentials for the CLI.
  * Required before every API call.
- * @param {Function} cb The callback
- * @param {boolean} isLocal If we should load local API credentials for this clasp project.
  */
-export function getAPICredentials(cb: (rc: ClaspSettings | void) => void) {
-    DOTFILE.RC_LOCAL.read().then((rc: ClaspSettings) => {
-      oauth2Client.setCredentials(rc);
-      cb(rc);
-    }).catch((err: any) => {
-      DOTFILE.RC.read().then((rc: ClaspSettings) => {
+export async function loadAPICredentials() {
+    return new Promise((resolve, reject) => {
+      DOTFILE.RC_LOCAL.read().then((rc: ClaspSettings) => {
         oauth2Client.setCredentials(rc);
-        cb(rc);
+        resolve();
       }).catch((err: any) => {
-        console.error('Could not read API credentials. Error:');
-        console.error(err);
-        process.exit(-1);
+        DOTFILE.RC.read().then((rc: ClaspSettings) => {
+          oauth2Client.setCredentials(rc);
+          resolve();
+        }).catch((err: any) => {
+          console.error('Could not read API credentials. Error:');
+          console.error(err);
+          reject();
+          process.exit(1);
```

Comment:
Related to above comment, if we reject, then should we still exit after?

Reply:
There's no need to reject a promise before a `process.exit()`.

I believe `resolve/reject` will just mark the function a certain way (fulfilled, pending, etc) before returning back to the caller. In this case it doesn't matter since we're killing the process and won't pop the function stack.